doc/user/content/sql/tail.md

```diff
@@ -195,6 +195,9 @@ It is `true` if no more timestamps will appear that are strictly less than the
 timestamp.
 All further columns after `progressed` will be `NULL` in the `true` case.
 
+`mz_progressed` can be used to distinguish between a result with that contains
+no data (`true`) and a timeout (no rows returned). 
```

Comment:
So this doesn't add anything that's not already covered here (though `progressed` is a typo now; I'll fix that in another PR). "a result" here is too imprecise to be useful, I think.

Certainly we should try to understand why this doc failed our users, but the next paragraph is already covers exactly this situation, IMO.

Reply:
Hmm, i see your point! I can close this PR and open a new one.  I wonder if it's a combination of a few things: 

1. we were all reading the doc really quickly, and were overly focused specifically on the "initial snapshot" scenario 
2. the `Without...it is impossible` phrasing caused my brain to skim over the section a bit